src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -4,138 +4,174 @@ description: Release notes for Nuxeo Drive.
+Welcome to the Release Notes for **Nuxeo Drive 4.4.5**
 
-**Status**: <font color="#0066ff">**Release**</font> </br>
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/4.4.4.md)
+**Status**: <font color="#f04">**Beta**</font> </br>
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/4.4.5.md)
 
 ## Important Changes
 
 ### Direct Transfer
 
-The Direct Transfer has been re-enabled as a BETA feature, it can be activated through the Features tab (see the following paragraph about the [Feature tab](#new-features-tab)).
+#### Folder Uploads
 
-The Direct Transfer feature [leverages the FileManager](https://jira.nuxeo.com/browse/NXDRIVE-2065) for files upload.
+Folders upload through the Direct Transfer feature has been enabled. The feature now creates folders via the [FileManager]({{page version='' space='nxdoc' page='file-manager'}}). Uploads of complex assets e.g. documents with deep hierarchies of folders and files are now well supported up to multiple thousands of documents.
 
-#### Transfer Window
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA ticket [NXDRIVE-2019](https://jira.nuxeo.com/browse/NXDRIVE-2019)
 
-Once a transfer is launched, a popup shows the list of all the files with their metrics (progress bar, file name, transferred data).
+#### Duplicates Behavior
 
-Direct Transfers can be paused, resumed and cancelled through this interface. A link to the remote folder is available in front of each file.
+You can now choose what to do when a transfer would create a duplicate document on the server. The setting is effective for all files that will be sent at the same time (it is called the session). Each session has its own duplicates behavior.
 
-![]({{file name='direct-transfer-popup.png'}})
+Available options are:
+- **Create**: a duplicate document will be created, this is the same behavior as when adding a new file from Web UI.
+- **Ignore**: the transfer will be cancelled, preventing the duplicate creation.
+- **Override**: the document will be replaced on the server.
 
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i>&nbsp;More on JIRA tickets [NXDRIVE-2208](https://jira.nuxeo.com/browse/NXDRIVE-2208), [NXDRIVE-1926](https://jira.nuxeo.com/browse/NXDRIVE-1926) and [NXDRIVE-2219](https://jira.nuxeo.com/browse/NXDRIVE-2219).
+![]({{file name='duplicates-behavior-dt.png' page='nuxeo-drive-release-notes'}})
 
-#### Duplicates Management
+{{#> callout type='note'}}
+The **Ignore** option may be problematic on huge trees: for each and every file, an NXQL query will be done on the server to verify the existence of a similar document.
+And that query is not using Elasticsearch, so if you have folderish documents with thousands of children, that query will likely be a bottleneck.
+{{/callout}}
```

Comment:
I thought about it and this note is not really necessary finally. WDYT @aescaffre?
```suggestion
```